src/bot/index.ts

```diff
@@ -126,7 +126,8 @@ bot.on(mSDK.RoomEvent.Timeline, (event: mSDK.MatrixEvent) => {
     logger.debug(`Processed receiver to address ${address} and parachain id ${parachain_id}`);
 
     try {
-      decodeAddress(address);
+      // TODO: check this
```

Comment:
What needs to be checked?

Reply:
I've removed it locally, but forgot to commit it :\
I wanted to doublecheck that it actually works as an address validation (it does)